make.js

```diff
@@ -345,8 +345,8 @@ target.testLegacy = function() {
     run('mocha ' + testsSpecPath, /*inheritStreams:*/true);
 }
 
-target.package = function() {
-    // validate powershell 5
+target.layout = function() {
```

Comment:
are you planning to skip the creating the aggregated layout if the current branch is not master and not a releases/* branch?